pkg/datagatherer/eks/eks.go

```diff
@@ -13,17 +15,22 @@ type EKSDataGatherer struct {
 	clusterName string
 }
 
+type EKSDataGathererConfig struct {
+	ClusterName string
+	DataPath    string
```

Comment:
I think `DataPath` should not be here.

Reply:
We can fix this later: https://github.com/jetstack/preflight/pull/82#discussion_r377745851